uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/io/HabitsCSVExporter.kt

```diff
@@ -124,7 +124,7 @@ class HabitsCSVExporter(
         val dateFormat = DateFormats.getCSVDateFormat()
         for ((timestamp, value) in entries.getKnown()) {
             val date = dateFormat.format(timestamp.toJavaDate())
-            out.write(String.format(Locale.US, "%s,%d\n", date, value))
+            out.write(String.format(Locale.US, "%s,%.3f\n", date, value.toDouble() / 1000.0))
```

Comment:
This is a quite dangerous move to use `%.3f` here. The better approach would be to use `opencsv` library, to a have a somewhat standardized approach with these things (the dot in fraction point might be some headache maker).

Reply:
... also the 1000 thing should be standardized somehow, cause it could be often forgotten using this value.